docs/components/tabs.md

```diff
@@ -192,166 +216,124 @@ Here's an example (code only) of utlizing a dynamic `router-view` component with
     </router-link>
   </template>
 </KTabs>
-<router-view
-  v-slot="{ route }"
->
+
+<router-view v-slot="{ route }">
   <h3>Router View content</h3>
   <p>{{ route.path }}{{ route.hash }}</p>
 </router-view>
 ```
 
 ## Slots
 
-### Anchor Content
+### anchor & panel
 
 The tab control defaults to the `tab.title` string. You may use the `#{tab.hash}-anchor` slot to customize the content of the tab control.
 
+In order provide the tab panel content (when the `hasPanels` prop is set to `true`) you must slot the content in the named slot, defined by the `tab.hash` string, without the `#`.
+
 <KTabs :tabs="slottedTabs">
-  <template #pictures-anchor>
-    Custom Pictures Tab
+  <template #gateway-anchor>
+    <KongIcon />
+    Gateway
   </template>
-  <template #pictures><p>Wow look <b>Pictures!</b></p></template>
-  <template #movies-anchor>
-    I love movies
+  <template #gateway><b>Gateway</b> tab content</template>
+  <template #notifications-anchor>
+    <InboxNotificationIcon />
+    Notifications
+    <KBadge appearance="decorative">3</KBadge>
   </template>
-  <template #movies><p>Wow look <b>Movies!</b></p></template>
-  <template #books-anchor>
-    Need a book?
+  <template #notifications><b>Notifications</b> tab content</template>
+  <template #docs-anchor>
+    <BookIcon />
+    Documentation
   </template>
-  <template #books><p>Wow look <b>Books!</b></p></template>
+  <template #docs><b>Documentation</b> tab content</template>
 </KTabs>
 
 ```html
-<template>
-  <KTabs :tabs="tabs">
-    <template #pictures-anchor>
-      Custom Pictures Tab
-    </template>
-    <template #pictures><p>Wow look <b>Pictures!</b></p></template>
-    <template #movies-anchor>
-      I love movies
-    </template>
-    <template #movies><p>Wow look <b>Movies!</b></p></template>
-    <template #books-anchor>
-      Need a book?
-    </template>
-    <template #books><p>Wow look <b>Books!</b></p></template>
-  </KTabs>
-</template>
-
-<script setup lang="ts">
-const tabs = [
-  { hash: '#pictures', title: 'Pictures' },
-  { hash: '#movies', title: 'Movies' },
-  { hash: '#books', title: 'Books' },
-]
-</script>
+<KTabs :tabs="tabs">
+  <template #gateway-anchor>
+    <KongIcon />
+    Gateway
+  </template>
+  <template #gateway><b>Gateway</b> tab content</template>
+  <template #notifications-anchor>
+    <InboxNotificationIcon />
+    Notifications
+    <KBadge appearance="decorative">3</KBadge>
+  </template>
+  <template #notifications><b>Notifications</b> tab content</template>
+  <template #docs-anchor>
+    <BookIcon />
+    Documentation
+  </template>
+  <template #docs><b>Documentation</b> tab content</template>
+</KTabs>
 ```
 
-### Panel Content
+## Events
 
-In order provide the tab panel content (when the `hasPanels` prop is set to `true`) you must slot the content in the named slot, defined by the `tab.hash` string, without the `#`.
+### changed
```

Comment:
Should the event be updated to `@change` instead so that it is present-tense, similar to `@click` etc.?